app/views/handlers/summary.py

```diff
@@ -15,4 +17,24 @@ def get_context(self):
         block = self._schema.get_block(self._current_location.block_id)
         collapsible = block.get("collapsible", False)
         context = questionnaire_summary_context(collapsible)
+
+        if submission_schema := self._schema.get_submission():
```

Comment:
You don't need the walrus, it can just be
```
submission_schema := self._schema.get_submission():
```
and then dump the else

Reply:
Thanks, I've moved the code now and added a test 👍